app/logic/Mail/ActiveSync/ActiveSyncAccount.ts

```diff
@@ -163,7 +163,7 @@ export class ActiveSyncAccount extends MailAccount {
     if (response.ok) {
       let versions = (response.MSASProtocolVersions || "").split(",");
       if (versions.includes("14.1")) {
-        this.protocolVersion = "14.1";
+        this.protocolVersion = versions.includes("16.1") ? "16.1" : "14.1";
```

Comment:
That's a little strange, given the `if` above, but alright...

Reply:
I need to be sure that the server supports both versions. I wrote that in my PR comment but I guess I should have put it in as a code comment as well...